src/viewProviders/schemas.ts

```diff
@@ -47,7 +47,7 @@ export class SchemasViewProvider implements vscode.TreeDataProvider<SchemasViewP
     ccloudConnected.event((connected: boolean) => {
       // TODO(shoup): check this for CCloud vs local once we start supporting local SR; check the
       // TopicViewProvider for a similar check
-      logger.debug("ccloudConnected event fired, resetting", connected);
+      logger.debug("ccloudConnected event fired, resetting", { connected });
```

Comment:
Minor formatting change since I couldn't remember what the `false` meant in:
```
ccloudConnected event fired, resetting false
```
So we should now see:
```
ccloudConnected event fired, resetting { connected: false }
```